cellxgene_schema_cli/cellxgene_schema/cli.py

```diff
@@ -77,6 +77,31 @@ def add_labels(input_file, output_file):
         sys.exit(1)
 
 
+@schema_cli.command(
+    name="validate-fragment",
```

Comment:
Since this can do more than just validation, I don't love the name. Maybe `"process-fragment"`?